avm/res/sql/instance-pool/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.28.1.47646",
-      "templateHash": "6377684067214410829"
+      "version": "0.31.92.45157",
```

Comment:
This will trigger module publishing but no update is done to the module functionality. I'd suggest to revert the main.json changes